src/config/config.js

```diff
@@ -41,7 +42,16 @@ export const fuseNetwork = {
   g$Decimals:2
 }
 
-const ethereum = {
+let altProviders = {}
```

Comment:
we have https://github.com/GoodDollar/GoodDAPP/blob/master/src/lib/utils/string.js#L23 doing exactly this thing (safe-parse)